packages/primitives/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Image } from 'theme-ui'
+import { SxStyleProp } from '@kodiak-ui/core'
+import { Box } from '../Box'
+
+export type AvatarProps = {
+  imageSrc?: string
+  text: string
+  sx?: SxStyleProp
+}
+
+export function Avatar({ imageSrc, text, sx = {} }: AvatarProps) {
+  return imageSrc ? (
+    <Image src={imageSrc} sx={{ borderRadius: 'full', ...sx }} alt={text} />
```

Comment:
What do you think about trying an `objectFit: cover` here to get the image to retain a bit of it's original proportions?